src/index.rs

```diff
@@ -261,11 +272,20 @@ impl Index {
         &mut outpoint_to_txid,
         &mut coinbase_inputs,
         &mut ordinal_ranges_written,
+        &mut outputs_in_block,
       )?;
     }
 
     height_to_hash.insert(&height, &block.block_hash())?;
 
+    // TODO: how to set default differently?
+    if outputs_traversed.is_empty()? {
+      outputs_traversed.insert("outputs_traversed", &0)?;
+    }
```

Comment:
I think we never need to set a default, as long as when we get it, we use zero if a value isn't present.

```suggestion
```